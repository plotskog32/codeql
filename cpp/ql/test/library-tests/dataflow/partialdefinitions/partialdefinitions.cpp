
struct MyStruct
{
	int x;
	struct MySubStruct {
		int z;
	} y;
};

void test()
{
	MyStruct s;

	s.x = 1;
	s.y.z = 1;
}
