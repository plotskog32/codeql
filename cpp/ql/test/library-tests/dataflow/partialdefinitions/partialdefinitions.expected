| partialdefinitions.cpp:14:2:14:2 | partial def of s | partialdefinitions.cpp:14:2:14:2 | s | partialdefinitions.cpp:14:2:14:8 | ... = ... |
| partialdefinitions.cpp:15:2:15:2 | partial def of s | partialdefinitions.cpp:15:2:15:2 | s | partialdefinitions.cpp:15:2:15:10 | ... = ... |
| partialdefinitions.cpp:15:4:15:4 | partial def of y | partialdefinitions.cpp:15:4:15:4 | y | partialdefinitions.cpp:15:2:15:10 | ... = ... |
